server/modules/modules.go

```diff
@@ -17,7 +20,19 @@ func EmptyType() {
 
 func Message(message *messageService.Message, messageType int, conn *websocket.Conn) {
 
-	err := coremessage.SendMessage(message, messageType)
+	isBlocked, err := settingService.IsUserBlocked(message)
+	if err != nil {
+		loger.Log.Errorf(" Error has occurred : ", err)
+		byteError, _ := json.Marshal(changeNicknameError) // this for UI
+		conn.WriteMessage(messageType, byteError)         // this for UI
+		return
+	}
+	if isBlocked {
+		loger.Log.Errorf("This contact is blocked")
+		return
+	}
+
+	err = coremessage.SendMessage(message, messageType)
```

Comment:
Cannot use `coremessage.SendMessage(message, messageType) (type messageService.Message)` as type `error` in assignment